ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -56,6 +61,48 @@ final class ApiConfigManagementService private (
       .map(_.fold(defaultConfigResponse) { case (_, conf) => configToResponse(conf) })(DE)
       .andThen(logger.logErrorsOnCall[GetTimeModelResponse])(DE)
 
+  /** After a short delay, this method checks if there exists a ledger configuration,
+    *  and submits a default configuration if no ledger configuration exists.
+    *
+    *  There are several reasons why the change could be rejected:
+    *  - The participant is not authorized to set the configuration
+    *  - There already is a configuration, it just didn't appear in the index yet
+    *  This method therefore does not try to re-submit the initial configuration in case of failure.
+    */
+  private def submitInitialConfig(initialConfig: Configuration) = {
+    implicit val executionContext: ExecutionContext = DE
+    Delayed.Future.by(
+      Duration.fromNanos(ledgerConfiguration.initialConfigurationSubmitDelay.toNanos))(
+      for {
+        optConfig <- index.lookupConfiguration()
+        _ <- if (optConfig.isDefined)
+          Future.successful(())
+        else {
+          val submissionId = SubmissionId.assertFromString(UUID.randomUUID.toString)
+          logger.info(
+            s"No ledger configuration found, submitting an initial configuration $submissionId")
+          FutureConverters
+            .toScala(
+              writeService.submitConfiguration(
+                Timestamp.assertFromInstant(timeProvider.getCurrentTime.plusSeconds(60)),
+                submissionId,
+                initialConfig
+              ))
+            .map {
+              case SubmissionResult.Acknowledged =>
+                logger.info(s"Initial configuration submission $submissionId was successful")
+                ()
+              case result =>
+                logger.warn(
```

Comment:
Can we lower the logging level to info? In distributed ledgers, the participant administrator are not necessarily authorized to define the time model (owned by the domain admin). In such cases submission would fail expectedly. In such cases a warning would indicate an incorrectly operating node which is not necessarily the case.

Reply:
Done. `NotSupported` will emit a message at info level.

Reply:
Thanks Robert. Much obliged.